official/recommendation/data_pipeline.py

```diff
@@ -261,6 +261,7 @@ def get_dataset(self, batch_size, epochs_between_evals):
 
       file_pattern = os.path.join(
           epoch_data_dir, rconst.SHARD_TEMPLATE.format("*"))
+      from tensorflow.contrib.tpu.python.tpu.datasets import StreamingFilesDataset
```

Comment:
Can you leave a TODO describing why this is the case? For either you or someone in the TPU team to fix later.

Reply:
Can we add a username to the TODO, as well as add an explanation of why do it this way?

Reply:
i dont see how this was resolved, the TODO looks the same as before to me.

Reply:
Must have missed this comment. Added my user name to the TODOs.